fetch.bs

```diff
@@ -127,6 +117,9 @@ specifications:
  <li>`<code>Referer</code>` [[!REFERRER]]
 </ul>
 
+<p>To do so it also supersedes the HTTP `<a http-header><code>Origin</code></a>` header semantics.
+[[ORIGIN]]
```

Comment:
Maybe "originally defined in _The Web Origin Concept_ or similar, to make it clear that [[ORIGIN]] is a reference to a superseded document, not a normal reference?

Reply:
Fair. I or someone else should probably do a separate pass to add `<cite>` to things at some point.